### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by EddySantos07**
remember to npm install

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -8,8 +8,15 @@ require('dotenv').config();
 app.use(morgan('dev'));
 app.use(express.json());
 
+
 app.use(express.static(path.join(__dirname, '..', 'dist')));
 
+app.get('*', (req, res) => {
```
Comment (nastassiafulconis) [inline server/index.js]: Hey Eddy!
Before I accept this pull request, could you change the * in this app.get to only accept requests from the specific routes? (what we talked about earlier today)

File: src/components/App.jsx [hunk 2]
```diff
@@ -1,13 +1,39 @@
-import React, {Fragment} from "react";
-import Example from '../containers/exampleCounterContainer.js';
+import React, { Fragment } from "react";
+import Example from "../containers/exampleCounterContainer.js";
 
-const App = ({name}) => {
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+
+import LandingPage from "./MainComponents/LandingPage_Components/LandingPage.jsx";
```
Comment (nastassiafulconis) [inline src/components/App.jsx]: I am going to accept the other pull requests first so you can change these to be Christina and Anton's components! Will let you know when I have done this :)

File: src/index.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import App from "./components/App.jsx";
 import { Provider } from 'react-redux';
 import {store} from './store.js';
 
-ReactDOM.render(
+ReactDOM.hydrate(
```
Comment (nastassiafulconis) [inline src/index.js]: What is .hydrate doing here? (just out of curiosity)

**Review by nastassiafulconis — COMMENTED** [review]
Hey Eddy!
Left you a couple comments. I'm going to review the other pull requests first so you can change React Router to work with Christina and Anton's components. Also please change the asterisk to be route specific. Thanks for the great work!

File: src/index.js (hunk 3 above)
Comment (EddySantos07) [reply src/index.js]: @nastassiafulconis my bad this was when i was testing server side rendering! ill update this pr and sorry im getting back to you so late!

**Comment by EddySantos07** [comment]
@nastassiafulconis please review updated pr ah shoot i forgot to tweak something again disregard this! give me a few

**Comment by EddySantos07** [comment]
@nastassiafulconis now you can review this pr

**Comment by EddySantos07** [comment]
@nastassiafulconis ITS **ready**!!

**Review by nastassiafulconis — APPROVED** [review]
Looks good! Thanks so much Eddy!

**Pull request merged**